Prace_domowe/Praca_domowa2/Adrian Stańdo/pd_02_finished.ipynb

```diff
@@ -0,0 +1,3118 @@
+{
```

Comment:
<p>Możesz wskazać taki przykład?</p><p>Imo "Odzież, Obuwie, Dodatki" to jak najbardziej jedna kategoria.</p><p>Jeśli nie masz racji, to są skopiowane kolumny</p><p> </p><p>Czemu "Odzież, obuwie, dodatki" jest złą *main_category*, a "Książki i Komiksy" już tak?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Nie masz racji</p><p> </p>